backend/src/middleware/passportStrategies.ts

```diff
@@ -0,0 +1,70 @@
+import { ExtractJwt, Strategy as JwtStrategy } from 'passport-jwt';
+import { Strategy as LocalStrategy } from 'passport-local';
+import passport from 'passport';
+import User from 'models/user';
+
+export const registerStrategy = new LocalStrategy(
+  {
+    usernameField: 'email',
+    passwordField: 'password',
+  },
+  async (email, password, done) => {
+    try {
+      if (await User.findOne({ where: { email } })) {
+        return done(new Error('User already exists'), null);
+      }
+
+      const user = await User.create({ email, password });
+
+      return done(null, user, { message: 'Signed up Successfully' });
+    } catch (error) {
+      return done(error);
+    }
+  },
+);
+
+export const loginStrategy = new LocalStrategy(
+  {
+    usernameField: 'email',
+    passwordField: 'password',
+  },
+  async (email, password, done) => {
```

Comment:
I saw a comment about types in @abasic review but I see you haven't fixed it. Leaving this here if you forgot to add type annotations, if it's intentional please resolve the comment (and explain on slack 😄 )